docs/components/radio.md

```diff
@@ -12,6 +12,13 @@ title: Radio
     >
 </component>
 
+<component 
```

Comment:
Maybe add good use cases for these?

Reply:
do any other components have use case examples? should that just be its own heading?

Reply:
yea fair point lol